tutorials/continuous-aggs-tutorial.md

```diff
@@ -129,14 +129,20 @@ We can view the metadata for the continuous aggregate in the
 **timescaledb_information.continuous_aggregates** view.
 
 ``` sql
-SELECT view_name, refresh_lag, refresh_interval, max_interval_per_job, materialization_hypertable
+SELECT view_name
+     , refresh_lag
```

Comment:
Is this a common syntax to have the comma preceding vs. comma at the end of the previous line?

Reply:
Both styles are common, I just pick this out of habit. We should have a common style in our documents and stick to it, so whatever is decided I will use.